source/whats-new.txt

```diff
@@ -43,6 +44,32 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.1:
+
+What's New in 6.1
+-----------------
+
+The {+driver-short+} v6.1 release includes the following features:
+
+- Raises the ``bson`` package version to 6.1.0 to expose the
+  ``Decimal128.fromStringWithRounding()`` method. To learn more, see the
+  `v6.1.0 bson release notes <https://github.com/mongodb/js-bson/releases/tag/v6.1.0>`__.
+
+- Detects environment variables for region settings when
+  authentication by using the ``MONGODB-AWS`` authentication mechanism.
+  The driver sets region options only if both of the following
+  environment variables are present:
```

Comment:
Suggestion:

I think it could be good to identify who is responsible for setting the environment variables. I also think it could be more clear to mention they need to be set rather than "present". The latter doesn't seem like a common way to describe environment variables since they are either set or not set.

Maybe a sentence with a different structure could present it more clearly:

```suggestion
   To instruct the driver to pass your region options, you must set both of the following environment variables:
```

I wonder if maybe this information might be easier to find if incorporated in the auth mechanism content, and replace this explanation with a link to that content.

Reply:
Creating a ticket for this